sound/soc/sof/loader.c

```diff
@@ -389,8 +390,14 @@ int snd_sof_run_firmware(struct snd_sof_dev *sdev)
 	}
 
 	mutex_lock(&sdev->cores_mask_mutex);
-	/* fw boot is complete. Update the active cores mask */
+	/* FW boot is completed, update the status of dsp cores */
```

Comment:
@keyonjie the PR looks good just a minor nit. You mean FW boot is complete, update the ref counts here right? status is not very intuitive.

Reply:
Thanks, let me correct it. I used 'status' with intention to include core_mask and ref counts.